youtube_dl/extractor/funimation.py

```diff
@@ -227,6 +249,81 @@ def _real_extract(self, url):
             'display_id': display_id,
             'title': title,
             'description': description,
+            'series': artist,
+            'season_id': season_id,
+            'season_number': season_number,
+            'episode_id': item.get('videoUrl'),
+            'episode': episode,
+            'episode_number': episode_number,
             'thumbnail': thumbnail,
             'formats': formats,
         }
+
+
+class FunimationShowPlaylistIE(FunimationBaseIE):
+    IE_NAME = 'funimation:playlist'
+    _VALID_URL = r'(https?://(?:www\.)?funimation\.com/shows/(?P<id>[^/]+))(?:/(?:home|about|videos))?$'
```

Comment:
Do you mean the unnamed capture that starts at "https?" and ends after the <id> capture? It's used below on [line 290](https://github.com/rg3/youtube-dl/pull/9515/files#diff-0284ffa7d6a90af242344617e3fc234fR290) to capture the series URL. I can add a name to the capture group to make that more clear.